### PR TITLE
Implement some sort of :tabbable selector. Closes #1

### LINKED ISSUE #1: Extend focusable detection
Inspiration: http://api.jqueryui.com/focusable-selector/

### PR DESCRIPTION
**Pull request by andrey-skl**

### CODE REVIEW

**Comment by andrey-skl** [comment]
Could you please merge and close it?

P.S. As I see you are not maintaining this for years. Could you please grant me developer access to this repository and npm package to let me look after this package if needed?

File: angular-tab-trap.js [hunk 1]
```diff
@@ -50,10 +50,14 @@
                     focusTrailer.css(buttonStyle);
 
                     children.on('focus', function () {
-                        var focusables = transclusionPoint.find('input');
+                        var potentiallyTabbableChilds = transclusionPoint[0].querySelectorAll('input, button, select, textarea, a[href], *[tabindex]');
```
Comment (atesgoral) [inline angular-tab-trap.js]: Nitpick on variable name: Childs -> Children?

File: angular-tab-trap.js [hunk 2]
```diff
@@ -50,10 +50,14 @@
...
 
-                        if (focusables.length) {
-                            focusables[0].focus();
+                        var tababbles = Array.prototype.filter.call(potentiallyTabbableChilds, function(item) {
```
Comment (atesgoral) [inline angular-tab-trap.js]: Nitpick on white space use, to match style of rest of the code: Please leave a space after function: `function (item)`

**Comment by atesgoral** [comment]
Thanks for this! The project is still active, but there were interestingly no issue reports / pull requests.

File: angular-tab-trap.js (hunk 2 above)
Comment (paracycle) [inline angular-tab-trap.js]: Also the variable name should be `tabbables` and _not_ `tababbles`.

File: angular-tab-trap.js (hunk 1 above)
Comment (andrey-skl) [inline angular-tab-trap.js]: Oh yeah, sure. Sorry, I'm not a native speaker.

File: angular-tab-trap.js (hunk 2 above)
Comment (andrey-skl) [inline angular-tab-trap.js]: Thank you! Will do. UPD: done

**Comment by atesgoral** [comment]
👍 Thanks!

**Pull request merged**